sda-commons-starter-web/src/test/resources/application.properties

```
@@ -1,4 +1,6 @@
 management.server.port=0
 wiremock.server.port=0
 # disabling tracing by default for tests
-management.tracing.enabled=false
\ No newline at end of file
+management.tracing.enabled=false
+management.endpoint.tasks.access=read_only
```

In the notes you linked it's `read-only`. But now I'm confused because I assume, that this is tested.

```suggestion
management.endpoint.tasks.access=read-only
```